packages/utils/src/index.js

```diff
@@ -4,6 +4,7 @@ export { default as DbNameValidator } from './DbNameValidator';
 export { default as DOMUtils } from './DOMUtils';
 export { default as Pending } from './Pending';
 export { default as PromiseUtils } from './PromiseUtils';
```

Comment:
Could also just do an `export class PromiseUtils` from PromiseUtils then don't need this line since next line will export it all

Reply:
Good call